extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -274,6 +274,12 @@ public static enum Source {
             @ConfigItem
             public Provider secretProvider = new Provider();
 
+            /**
+             * The string representation of the private key content.
```

Comment:
```suggestion
             * String representation of a private key. If provided, indicates that JWT is signed using a private key in PEM or JWK format.             
```

As far as I recall, the Doc team suggested `If provided, ` and this is also similar how other properties are documented.

And then add one more doc line, '*You can use the {@link #signatureAlgorithm} property to override the default key algorithm, `RS256`.'

This is the last suggestion, please squash afterwards and open the PR for review for the builds to run

Thanks